src/nanoemoji/glyph_reuse.py

```diff
@@ -26,66 +26,57 @@
 from .fixed import fixed_safe
 
 
-class ReuseResult(NamedTuple):
-    glyph_name: str
-    transform: Affine2D
-
-
-class GlyphReuseCache:
-    def __init__(self, reuse_tolerance: float):
-        self._reuse_tolerance = reuse_tolerance
-        self._known_glyphs = set()
-        self._reusable_paths = {}
+class ReuseResult:
+    pass
 
-        # normalize tries to remap first two significant vectors to [1 0], [0 1]
-        # reuse tolerence is relative to viewbox, which is typically much larger
-        # than the space normalize operates in. TODO: better default.
-        self._normalize_tolerance = self._reuse_tolerance / 10
-
-    def try_reuse(self, path: str) -> Optional[ReuseResult]:
-        """Try to reproduce path as the transformation of another glyph.
-
-        Path is expected to be in font units.
-
-        Returns (glyph name, transform) if possible, None if not.
-        """
-        assert (
-            not path in self._known_glyphs
-        ), f"{path} isn't a path, it's a glyph name we've seen before"
-        assert path.startswith("M"), f"{path} doesn't look like a path"
 
-        if self._reuse_tolerance == -1:
-            return None
-
-        norm_path = normalize(SVGPath(d=path), self._normalize_tolerance).d
-        if norm_path not in self._reusable_paths:
-            return None
+class GlyphInstruction(NamedTuple):
```

Comment:
when I read "glyph instructions" in the context of fonts, I always think about TrueType hinting, which is of course not the case here. Could we find a different name maybe? E.g. `ReuseResult` or `ReusedGlyph` or `GlyphReuse`

Reply:
Oh yeah good call, I had in mind "how to draw" but the overlap with hinting is very real

Reply:
Type removed